### PR TITLE
asadiqbal08/Removed course header from Learning MFE

### PR DESCRIPTION
**Pull request by asadiqbal08**
As per some [suggestions](https://github.com/openedx/build-test-release-wg/issues/81#issuecomment-928050901) by @davidjoy , We need to refactor the code in EdX Learning MFE to utlize the header from `frontend-component-header` instead of [course-header](https://github.com/edx/frontend-app-learning/tree/master/src/course-header)

[Fixes-263](https://github.com/mitodl/mitxonline/issues/263) 

#### Blocked By:
- [issue-252](https://github.com/mitodl/mitxonline/issues/252)
- [PR-133](https://github.com/edx/frontend-component-header/pull/133)

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @asadiqbal08! I've created [OSPR-6191](https://openedx.atlassian.net/browse/OSPR-6191) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by natabene** [comment]
@asadiqbal08 Thank you for your contribution. Please let me know once it is ready for our review.

File: src/tab-page/TabPage.jsx [hunk 1]
```diff
@@ -6,8 +6,7 @@ import { Redirect } from 'react-router';
 
 import Footer from '@edx/frontend-component-footer';
 import { Toast } from '@edx/paragon';
-import { Header } from '../course-header';
-import { getAccessDeniedRedirectUrl } from '../shared/access';
```
Comment (davidjoy) [inline src/tab-page/TabPage.jsx:10]: Looks like you deleted `getAccessDeniedRedirectUrl` here.  I see it used down on line 54.

**Comment by davidjoy** [comment]
This looks great - so I think we're going to need to get the frontend-component-header-edx portion of this done before I'd feel comfortable merging.

File: src/tab-page/TabPage.jsx (hunk 1 above)
Comment (asadiqbal08) [reply src/tab-page/TabPage.jsx:10]: Thanks @davidjoy for pointing out. The PR, I marked it as `WIP` as I am thinking to update it again after the merge of [PR#133](https://github.com/edx/frontend-component-header/pull/133)

**Comment by asadiqbal08** [comment]
@davidjoy should I created a corresponding PR to `frontend-component-header-edx` or someone else is taking care of it ?

**Comment by asadiqbal08** [comment]
@davidjoy another PR for your [reference](https://github.com/edx/frontend-component-header-edx/pull/196)

**Review by mikix — APPROVED** [review]
Thank you!

**Comment by mikix** [comment]
Looks like tests pass, but can you fix your commit header to be like: `feat: removed course header from Learning MFE` or something to satisfy our commit linter?

**Comment by asadiqbal08** [comment]
@mikix @davidjoy I was rebasing this PR with latest and came to know that a [recent PR -- engage product tour](https://github.com/edx/frontend-app-learning/pull/750/files) has merged that contains some changes in components of `course-header` that is going to be removed from this repository with this PR. 
 Please take a look on the change set and let me know what to do ?
- Should I need to create corresponding PRs in `frontend-component-header` and `frontend-component-header-edx` to keep the changes of product tour
- OR 
- Should We request the author of [PR -- engage product tour](https://github.com/edx/frontend-app-learning/pull/750/files) to move logic in LearningHeader of those repositories and revert those from this repo ?

**Comment by ciduarte** [comment]
@asadiqbal08 I just put up a [PR to remove tour logic from the Header component](https://github.com/edx/frontend-app-learning/pull/775)! Sorry for the mixup, you should be able to rebase and carry on when it's merged.

edit: Just merged!

**Comment by openedx-webhooks** [comment]
@asadiqbal08 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/frontend-app-learning&entry.867055334=https://github.com/edx/frontend-app-learning/pull/715&entry.1484655318=https://github.com/asadiqbal08&entry.752974735=2021-10-29+11:17&entry.1917517419=2021-12-20+13:57&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**